src/app/modules/item/pages/item-by-id/item-by-id.component.ts

```diff
@@ -195,7 +195,7 @@ export class ItemByIdComponent implements OnDestroy {
       })
     ).subscribe({
       next: itemRoute => this.itemRouter.navigateTo(itemRoute, { navExtras: { replaceUrl: true } }),
-      error: err => {
+      error: (err: Error) => {
```

Comment:
How do you know for sure it is a `Error`, I don't think you can do this assumption.
Isn't related to https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-4.html#defaulting-to-the-unknown-type-in-catch-variables---useunknownincatchvariables ?

Reply:
fixed